kotskinds/apis/kots/v1beta1/license_types.go

```diff
@@ -114,7 +114,7 @@ type LicenseSpec struct {
 	IsIdentityServiceSupported     bool                        `json:"isIdentityServiceSupported,omitempty"`
 	IsGeoaxisSupported             bool                        `json:"isGeoaxisSupported,omitempty"`
 	IsSnapshotSupported            bool                        `json:"isSnapshotSupported,omitempty"`
-	IsNewKOTSUIEnabled             bool                        `json:"isNewKotsUiEnabled,omitempty"`
+	IsNewKOTSUIEnabled             bool                        `json:"isNewKotsUiEnabled,omitempty"` // 2022-02-17: Unused, but kept for backwards compatibility
```

Comment:
why? removing this won't break anything, and it's not gonna be used anywhere.

Reply:
I mistakenly thought unmarshalling without these tags would be an error. Updated.